src/App.jsx

```diff
@@ -0,0 +1,23 @@
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import AppLayout from './components/AppLayout';
+import About from './pages/About';
+import Home from './pages/Home';
+import Van from './pages/vans/Van';
+import Vans from './pages//vans/Vans';
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+      <Route element={<AppLayout/>}>
+        <Route path="/" element={<Home />} />
+        <Route path="/about" element={<About />} />
+        <Route path='/vans' element={<Vans />} />
+        <Route path='/vans/:id' element={<Van />} />
+      </Route>
+        
```

Comment:
Remove this space is not needed because the main reason is to help your app run more efficiently.